src/dom.js

```diff
@@ -443,15 +443,16 @@ type PopupOptions = {|
     resizable? : 0 | 1,
     toolbar? : 0 | 1,
     menubar? : 0 | 1,
-    scrollbars? : 0 | 1
+    scrollbars? : 0 | 1,
+    closeOnUnload? : 0 | 1
 |};
 
 export function popup(url : string, options? : PopupOptions) : CrossDomainWindowType {
 
     // $FlowFixMe
     options = options || {};
 
-    const { width, height } = options;
+    const { width, height, closeOnUnload = 1 } = options;
```

Comment:
Can we pull out `...restOptions` here? Otherwise this will be passed to window.open and may cause problems.

Reply:
@bluepnume Looks like we are already doing that down further: https://github.com/krakenjs/belter/blob/6c6c620693c400b000fd5b6662d54f587115ed5b/src/dom.js#L477